docs/host-your-own.md

```diff
@@ -397,9 +397,26 @@ Now point your browser to `http://localhost:8080`.
 - To stop the containers run `docker-compose stop`.
 - To debug the NodeJS server run `docker logs pairdrop_node_1`.
 
-
 <br>
 
+# Coturn
+## docker-compose
+
+- copy `docker-compose-coturn_example.yml` to `docker-compose-coturn.yml`
+- copy `rtc_config_example-coturn.json` to `rtc_config.json`
+- copy `turnserver_example.conf` to `turnserver.conf`
+- change <DOMAIN> in all three files to the domain, where your pairdrop is running
+- change user and password for turn-server in `turnserver.conf` and `rtc-config.json`
```

Comment:
Probably, there should be another step that you need to generate ssl certificates

Reply:
I changed this to the new improvements and added this step.